### PR TITLE
Feat: Add Pushbullet notification channel

### LINKED ISSUE #942: Use Pushbullet as a notification channel
It would be very cool to add Pushbullet as a notification channel, especially when they opened their API:
https://docs.pushbullet.com/

### PR DESCRIPTION
**Pull request by dennypradipta**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
This PR closes #942 

## How did you implement / how did you fix it  
- Added the notification channel
- Added the validators
- Added tests

## How to test  
1. Use this config:
```                                 
notifications:
  - id: pushbullet
    type: pushbullet
    data:
      token: <access_token>
      deviceID: <device_iden> // optional
probes:
  - id: '1'
    name: URI without credential
    requests:
      - url: https://httpbin.org/delay/{{ number 0 2 }}
    alerts:
      - assertion: response.time > 1000
        message: Not quite my tempo
    incidentThreshold: 2
    recoveryThreshold: 2
```
2. Run

image

### CODE REVIEW

**Review by kevin-hyperjump — APPROVED** [review]

File: src/components/config/validate.ts [hunk 1]
```diff
@@ -118,6 +118,11 @@ const PUSHOVER_NO_USER = setInvalidResponse('USER not found')
 const GOTIFY_NO_TOKEN = setInvalidResponse('TOKEN not found')
 const GOTIFY_NO_URL = setInvalidResponse('URL not found')
 
+// Pushbullet
+const PUSHBULLET_NO_TOKEN = setInvalidResponse(
+  'Token not found! You can create your Access Token at https://www.pushbullet.com/#settings'
```
Comment (nicnocquee) [inline src/components/config/validate.ts:123]: Maybe better say that it's "Pushbullet token not found!" to make it clearer.

**Review by nicnocquee — CHANGES_REQUESTED** [review]
Missing documentation.

File: src/components/config/validate.ts (hunk 1 above)
Comment (dennypradipta) [reply src/components/config/validate.ts]: Fixed

**Comment by dennypradipta** [comment]
> Missing documentation.

Fixed

**Review by sapiderman — APPROVED** [review]
Looks good. 
🍜 🍝 🍳

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**